### ISSUE TITLE
Feature Request: Ability to view plain text or HTML of email without looking at raw MIME

### ISSUE THREAD

**Issue by GM-Polyakov** [nylas/N1#2987]

**Comment by grinich** [comment]
Does another mail app have this?

**Comment by GM-Polyakov** [comment]
Gmail and Outlook

**Comment by grinich** [comment]
Is this different from the "view source" thing in Gmail? Isn't that just the original MIME object?

**Comment by GM-Polyakov** [comment]
Yes -- https://support.office.com/en-us/article/Change-the-message-format-to-HTML-Rich-Text-or-plain-text-de2acb3d-3330-42a1-b02a-5f582fc6e796

**Comment by grinich** [comment]
@GM-Polyakov that link seems to show sending plain text vs html but not viewing plain text vs html. Are you looking to view plain vs. html or see the HTML source itself?

**Comment by GM-Polyakov** [comment]
Ah yeah, wrong link. Goal is to see the HTML source itself -- [like here](http://stackoverflow.com/questions/38058596/how-do-i-view-the-html-source-of-an-email-in-outlook-2016). Plain text would just be plain text though.

Use case is for examining or troubleshooting marketing emails.

**Comment by manicmarvin** [comment]
It's also linked to #2122, Airmail has this feature as well as the ability to edit HTML of emails.
